tools/mo/unit_tests/mo/convert/import_from_mo_test.py

```diff
@@ -86,7 +85,7 @@ def create_model_ref():
     ])
     # Checks convert import from openvino.tools.mo
     def test_import(self, params):
-        from openvino.tools.mo import convert_model
+        from openvino.runtime import convert_model
```

Comment:
Why do we test ovc `convert_model` here? Can we move this file to OVC unit-tests?

Reply:
This change was done before I added unit tests to ovc, so I can remove this change now.